kingdom.lua

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env tarantool
+
+local log = require('log')
+local fio = require('fio')
+local clock = require('clock')
+local fiber = require('fiber')
+local checks = require('checks')
+
+local work_dir = os.getenv('TARANTOOL_WORKDIR')
+local ok, err = fio.mktree(work_dir)
+if not ok then error(err, 0) end
+
+local LOCK_DELAY = os.getenv('TARANTOOL_LOCK_DELAY') or 10
+function _G.get_lock_delay()
+    return LOCK_DELAY
+end
+
+box.cfg({
+    work_dir = work_dir,
+    listen = os.getenv('TARANTOOL_LISTEN'),
+})
+
+box.schema.user.grant('guest', 'super', nil, nil, {if_not_exists = true})
```

Comment:
You have a limited number of functions/spaces here. Let's grant explicitly access to each space/function.

Do I assume correctly that everybody could connect to such instance and "eval" "os.execute"?